### ISSUE TITLE
Speed up device group list fetch

### ISSUE THREAD

**Issue by stellanl** [akvo/akvo-flow#2199]
The third part of #2035.
/rest/device_groups is still using the DtoMarshaller method.
Switching to beanutils should speed it up.

**Comment by stellanl** [comment]
Marshaling to dto is straightforward, but the beanutils-optimised to-domain code looks like this:
```
  // copy the properties, except the createdDateTime property,
  // because it is set in the Dao.
  BeanUtils.copyProperties(deviceGroupDto, s,
        new String[] {
         "createdDateTime"
        });
```
which makes me wonder if that should be the pattern for all entity creation.
It is not for (at least) device entites.

**Comment by janagombitova** [comment]
Is this issue also merged to UAT1 and will be released with 1.9.21? If yes, I can confirm everything around the devices works as expected. 

This bug is still there https://github.com/akvo/akvo-flow/issues/402 but that is a separate issue
